themes/qgrid/source/css/index.scss

```diff
@@ -1,217 +1,435 @@
+		text-align: center;
+		color: $color-top-text;
+		padding: 40px 0 0 0;
+
+		.qgrid-logo {
+			margin-top: 20px;
+			margin-bottom: 40px;
+			background-size: contain;
+		}
+
+
+		.frameworks {
+			list-style: none;
+			padding: 0;
+			margin: 40px 0 40px 0;
+			display: flex;
+			flex-flow: row wrap;
+			justify-content: center;
+			align-content: center;
+
+			li {
+				display: inline-block;
+				vertical-align: middle;
+				margin-right: 32px;
+				font-size: 5.2em;
+
+				&>a {
+					text-decoration: none;
+					color: white;
+				}
+
+				&:last-child {
+					margin-right: 0;
+				}
+
+				span {
+					color: white;
+				}
+
+				label {
+					display: block;
+					margin-top: 15px;
+					padding: 8px 0px;
+					border-radius: 4px;
+					width: 163px;
+
+					font: {
+						size: 18px;
+						family: $font-sans;
+						font-weight: normal;
+						text-transform: capitalize;
+					}
+
+					line-height: 24px;
+					background-color: #0078CF;
+					color: $whitescale-50;
+				}
+
+				&.ready label {
+					color: white;
+					cursor: pointer;
+				}
+
+				&.ready:hover label {
+					background-color: darken(#0078CF, 5%);
+				}
+			}
+		}
+	}
+
+	.example {
+		/* 	margin-top: -75px; */
+		align-items: center;
+		/* 	margin-bottom: 70px; */
+		display: flex;
+		flex-direction: column;
+	}
+
+	.go-to {
+		text-align: center;
+		margin-bottom: 100px;
+		font-size: 0;
+		display: flex;
+		flex-flow: row wrap;
+		justify-content: center;
+
+		a {
+			@extend %accent-button;
+			margin: 5px;
+		}
+	}
+}
+
+
+
+
+.news {
+	background-color: $color-background;
+	/* margin-bottom: 60px; */
+
+	&-wrapper {
+		display: flex;
+		justify-content: space-around;
+	}
+
+	&-heading {
+		text-align: center;
+		padding-bottom: 15px;
+
+		&__header2 {
+			margin-top: 20px !important;
+			margin-bottom: 40px !important;
+			font-weight: normal !important;
+			font-size: 2.142em !important;
+			line-height: 2.142em !important;
+			margin-bottom: 10px !important;
+			/* todo: refac to mixin h2 */
+		}
+	}
+
+	&-block {
+		width: 320px;
+		max-height: 400px;
+		background: $color-top-text;
+		border: 1px solid $color-border;
+		box-sizing: border-box;
+		border-radius: 4px;
+
+
+		&__img {
+			width: 100%;
+		}
+
+		&-text {
+
+			&__date {
+				margin-bottom: 0px;
+				font-family: $font-sans;
+				font-size: 1.142em;
+				text-align: left;
+			}
+
+			&__header3 {
+				margin-top: 8px;
+				font-size: 20px;
+				line-height: 26px;
+				font-weight: 400;
+			}
+
+			&__descr {
+				font-size: 16px;
+			}
+		}
+	}
+
+	&-button {
+		margin-top: 40px;
+		text-align: center;
+
+		&__btn {
+			@extend .button;
+			margin-top: 40px;
+			width: 144px;
+		}
+	}
+
+}
+
+.reasons {
+	padding-bottom: 60px;
+	background-color: $color-sidebar-background;
+	color: $color-top-text;
+
+	&-wrapper {
+		display: flex;
+		justify-content: space-around;
+	}
+
+	&-heading {
+		text-align: center;
+		padding-bottom: 15px;
+
+		&__header2 {}
```

Comment:
Empty block of css {} here and below 
Is this a reminder of something, or is it just a mistake?